src/charmed_kubeflow_chisme/juju/_juju.py

```diff
@@ -0,0 +1,41 @@
+# Copyright 2022 Canonical Ltd.
+# See LICENSE file for licensing details.
+"""Helper for shelling out to the Juju CLI"""
+
+from subprocess import Popen, PIPE
+from ruamel.yaml import YAML
+
+
+class Juju:
+    @staticmethod
+    def juju(*args, raise_on_stderr: bool=False):
+        cmd = ["juju"] + list(args)
+        proc = Popen(cmd, stdout=PIPE, stderr=PIPE)
+        stdout = proc.stdout.read().decode('utf-8')
+        stderr = proc.stderr.read().decode('utf-8')
+        if raise_on_stderr and stderr:
+            raise ValueError(f"failed to run juju command successfully.  Got this from stderr: {stderr}")
+        return stdout, stderr
+
+    @classmethod
+    def info(cls, charm_name: str):
```

Comment:
This one I love :D .... its classmethod so we can call staticmethod. Just wondering if it needs to be placed in juju class. Maybe separate functions? (not critical comment).

Reply:
lol yeah thinking back I'm not sure why I did it this way.  I guess I was channelling my inner java that day.  I think I copied part of this from somewhere and it was class based, then I just kept going without thinking.  Do you think the API would be better as functions?  

Thinking about it now, I think it would.  The class-based version I copied was written inline in a script somewhere, so having the class to group the commands together made sense.  Here, we've already for the package saying "`charmed_kubeflow_chisme.juju` is a bunch of juju tools", adding the extra class doesn't add value afaict

Reply:
Yeah my inner 'functional based everything Michal' self says it would haha in this usecase.

Reply:
Refactored it - what do you think now?